src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -191,14 +191,14 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 12,
-    color: COLORS.white2
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change is intentional and does not affect the readability or visibility of the text on different screens or under different lighting conditions.
```diff
-    color: COLORS.white2
+    color: COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->